code/modules/cm_preds/yaut_crafting.dm

```diff
@@ -0,0 +1,65 @@
+/// This .dm will contain all yautja craftable items and materials
+
+
+//=================//\\=================//
+/*
+				MATERIALS
+*/
+//=================\\//=================//
+/obj/item/xeno_tail
+	name = "Xenomorph Tail"
+	desc = "The tail of a terrible creature."
+	icon_state = "xeno_tail"
+//=================//\\=================//
+/*
+				Crafting
+*/
+//=================\\//=================//
+/obj/item/xeno_tail/attackby(obj/item/I, mob/user as mob)
+	..()
+	var/obj/item/stack/rods/stick = I
+	if(!HAS_TRAIT(user, TRAIT_SUPER_STRONG)) // Humans aint crafting these
+		to_chat(user,  SPAN_WARNING("You burn your hand while trying to manipulate the [src].breaking it in the process!"))
+		user.emote("scream")
+		qdel(src)
+	if(do_after(user, 3 SECONDS, INTERRUPT_NO_NEEDHAND, BUSY_ICON_BUILD, user))
+		to_chat(user,  SPAN_NOTICE("You try to couple the rod with the [src]."))
+		if(stick.use(1))
+			var/obj/item/weapon/melee/twohanded/yautja/spear/makeshift/incomplete/spear = new /obj/item/weapon/melee/twohanded/yautja/spear/makeshift/incomplete
+			user.put_in_hands(spear)
+			qdel(src)
+
+/obj/item/weapon/melee/twohanded/yautja/spear/makeshift/incomplete/attackby(obj/item/I, mob/user as mob)
```

Comment:
overall, I don't see the need for this being two different "intrim" items, it could very easily be one with two intermediate stages

Reply:
changed but i think it looked better the way i was doing it previously